controller/model/config_type_model.go

```diff
@@ -0,0 +1,62 @@
+/*
+	Copyright 2020 Netfoundry, Inc.
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+	https://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+*/
+
+package model
+
+import (
+	"fmt"
+	"github.com/netfoundry/ziti-edge/controller/persistence"
+	"github.com/netfoundry/ziti-foundation/storage/boltz"
+	"github.com/pkg/errors"
+	"go.etcd.io/bbolt"
+	"reflect"
+)
+
+type ConfigType struct {
+	BaseModelEntityImpl
+	Name string
+	Type string
+	Data map[string]interface{}
+}
+
+func (entity *ConfigType) ToBoltEntityForCreate(_ *bbolt.Tx, _ Handler) (persistence.BaseEdgeEntity, error) {
+	if entity.Name == CONFIG_TYPE_ALL {
```

Comment:
is this input trimmed before being compared?

Reply:
No, it's whatever is in the data store. We might want to put something in the backlog to consider if we want to handle trimming in the API or model layers. 

This will at least be handled consistently in that it won't be compared to trimmed values later.